### ISSUE TITLE
Visual Studio 2019 required to build ?

### ISSUE THREAD

**Issue by oudoulj** [reactjs/React.NET#834]
I've just cloned the project locally within Visual Studio **2017** and run dev-build.bat in a cmd prompt
```
C:\Users\jerome.oudoul\Source\Repos\React.NET2>dev-build.bat

C:\Users\jerome.oudoul\Source\Repos\React.NET2>call runs-msbuild.bat "/p:BuildType=Dev"
Could not find MSBuild.exe. Make sure Visual Studio 2019 is installed and try again.
```
In runs-msbuild.bat , I see mentions of VS **2019**
```
"%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\MSBuild.exe"
    "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\MSBuild.exe"
    "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Professional\MSBuild\Current\Bin\MSBuild.exe"
    "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\MSBuild.exe"
```

**Is Visual Studio 2019 required to build the ReactJS.NET project ?** 🤔

**Comment by oudoulj** [comment]
I ended up installing Visual Studio 2019 Community
I cloned the git and ran dev-build.bat in a cmd prompt as required.
It failed at home stating I was missing the .NET Core SDK although I had the latest one installed (2.2.300) so I gave up.
It worked at work though and the bat downloaded a billion things for a couple minutes.
I was then able to open the React.sln and build/rebuild the solution.
Set the React.Sample.Webpack.CoreMvc as my startup project and was able to launch it ! 😌 
![startup](https://i.ibb.co/bKSn5wC/startupscreen.png)

FYI here are the SDKs/FrameworksI have on my workstation on which I was able to run all this (some of them might have been installed along the VS2019 installation) :
![sdk](https://i.ibb.co/QpFK2qt/sdkatwork.png)
